source/retryable-writes/tests/README.rst

```diff
@@ -395,6 +395,57 @@ and sharded clusters.
 
    9. Disable the failpoint.
 
+#. Test that drivers return the original error after encountering an error with
+   label NoWritesPerformed. This test MUST be implemented by any driver that
+   implements the Command Monitoring specification. This prose test requires
+   drivers to set a fail point after an ``insertOne`` operation but before the
+   subsequent retry. Drivers that are unable to set a failCommand after the
+   CommandFailedEvent SHOULD use mocking or write a unit test to cover the same
+   sequence of events.
+
+   1. Create a client with ``retryableWrites=true``. If testing against a
+      sharded deployment, the test runner MUST ensure that the client connects
+      to only a single mongos host.
+
+   2. Configure a fail point with error code ``10107`` (NotWritablePrimary)::
```

Comment:
Would we be able to change this first error to a retryable write concern error so that we match Matthew's repro given in: https://jira.mongodb.org/browse/SERVER-66116?focusedCommentId=4526619&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4526619

Reply:
Absolutely!